.idea/ModFlowTool.iml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please do not commit the .idea folder to the project.
You can add this folder to your global gitignore.